redwood-treehouse-host/src/androidMain/kotlin/app/cash/redwood/treehouse/TreehouseLayout.kt

```diff
@@ -57,6 +57,12 @@ public class TreehouseLayout(
 
   override var saveCallback: TreehouseView.SaveCallback? = null
 
+  init {
+    // The view needs to have an id for Android to populate saved data back
+    @SuppressLint("ResourceType")
+    id = 9000
```

Comment:
This is really unsafe. You can put an `<item type="id">treehouse_layout</item>` into resources and use it via `R.id.treehouse_layout` so that the build system guarantees a unique ID value.

Reply:
You will need to turn on resources for this module as they're disabled by default.

https://github.com/cashapp/redwood/blob/89750b170fe3e8987227271dba45353d7bbc83d4/redwood-layout-view/build.gradle#L33-L36